azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -125,8 +129,11 @@
   private static final String AZKABAN_PRINCIPAL = "proxy.user";
   private static final String OBTAIN_JOBHISTORYSERVER_TOKEN =
       "obtain.jobhistoryserver.token";
+  private static final String OBTAIN_HIVESERVER2_TOKEN =
```

Comment:
This is a key in the job config file, right?

If so, please move it to the Constant class. 

New key names should follow the convention: use . to separate name spaces only. use _ to separate words in one namespace. See recently added new keys for examples.